samples/README.md

```diff
@@ -0,0 +1,52 @@
+[//]: # "This README.md file is auto-generated, all changes to this file will be lost."
+[//]: # "To regenerate it, use `python -m synthtool`."
+<img src="https://avatars2.githubusercontent.com/u/2810941?v=3&s=96" alt="Google Cloud Platform logo" title="Google Cloud Platform" align="right" height="96" width="96"/>
+
+# [Google Cloud Security Command Center: Node.js Samples](https://github.com/googleapis/nodejs-security-center)
+
+[![Open in Cloud Shell][shell_img]][shell_link]
+
+> Node.js idiomatic client for [Cloud Storage][product-docs].
+
+[Cloud Security Command Center](https://cloud.google.com/security-command-center/docs/) helps
+security teams gather data, identify threats, and act on them before they result in business
+damage or loss. It offers deep insight into application and data risk so that you can quickly
+mitigate threats to your cloud resources and evaluate overall health. With Cloud Security
+Command Center, you can view and monitor an inventory of your cloud assets, scan storage
+systems for sensitive data, detect common web vulnerabilities, and review access rights
+to your critical resources, all from a single, centralized dashboard.
+
+## Table of Contents
+
+* [Before you begin](#before-you-begin)
+* [Samples](#samples)
+  * [Quickstart](#quickstart)
+
+## Before you begin
+
+Before running the samples, make sure you've followed the steps outlined in
+[Using the client library](https://github.com/googleapis/nodejs-security-center#using-the-client-library).
+
+## Samples
+
+
+
+### Quickstart
+
+View the [source code](https://github.com/googleapis/nodejs-security-center/blob/master/samples/quickstart.js).
+
+[![Open in Cloud Shell][shell_img]](https://console.cloud.google.com/cloudshell/open?git_repo=https://github.com/googleapis/nodejs-security-center&page=editor&open_in_editor=samples/quickstart.js,samples/README.md)
+
+__Usage:__
+
+
+`node quickstart.js`
```

Comment:
If you're so inclined, might be worth adding:

```
// sample-metadata:
//  usage: node quickstart.js [organization]
```

So that folks know what argument to pass to the `quickstart.js` script, but definitely not a blocker for me.